src/playground/services/invoices/retrieve/endpoint.clj

```diff
@@ -27,3 +27,11 @@
     (if record
       (ring-resp/response (view/update-invoice request record))
       {:status 404 :body "Entry not in DB"})))
+
+(defn return-authored [{{:keys [id]} :path-params :keys [db] :as request}]
```

Comment:
avoid the word "return". Most of defns return something. Better: `get-author`, `author-of`